packages/client/hmi-client/src/components/workflow/ops/dataset-transformer/tera-dataset-transformer-node.vue

```diff
@@ -3,7 +3,17 @@
 		<tera-operator-placeholder v-if="!node.inputs[0].value" :node="node">
 			Attach one or more resources
 		</tera-operator-placeholder>
-		<Button v-else label="Edit" @click="emit('open-drilldown')" severity="secondary" outlined />
+		<template v-if="node.inputs[0].value">
+			<template v-if="!isEmpty(node.state.selectedOutputs)">
+				<tera-beaker-code-cell-output
+					v-for="m in node.state.selectedOutputs?.[0]?.messages"
+					:key="m.header.msg_id"
+					:jupyter-message="m"
+					:feature-config="{ isPreview: true }"
+				/>
+			</template>
```

Comment:
```suggestion
				<tera-beaker-code-cell-output
				  v-if="!isEmpty(node.state.selectedOutputs)"
					v-for="m in node.state.selectedOutputs?.[0]?.messages"
					:key="m.header.msg_id"
					:jupyter-message="m"
					:feature-config="{ isPreview: true }"
				/>
```

Reply:
I think using `v-if` and `v-for` on the same component is not recommended by Vue which is why I had the `template` tag there

Reply:
Why not do `v-for="m in (node.state.selectedOutputs?.[0]?.messages ?? [])" then and avoid the v-if?

Reply:
that works, I changed this up in my latest commit